src/management-system-v2/components/process-list.tsx

```diff
@@ -45,6 +54,37 @@ function folderAwareSort(sortFunction: (a: ProcessListProcess, b: ProcessListPro
   return sorter;
 }
 
+export function ProcessListItemName({
+  item,
+  divProps = {},
+}: {
+  item: { type: string; name: ReactNode };
+  divProps?: HTMLAttributes<HTMLDivElement>;
+}) {
+  const breakpoint = Grid.useBreakpoint();
+
+  return (
+    <div
+      className={
+        breakpoint.xs
+          ? styles.MobileTitleTruncation
+          : breakpoint.xl
+            ? styles.TitleTruncation
+            : styles.TabletTitleTruncation
+      }
+      {...divProps}
+      style={{
+        overflow: 'hidden',
+        whiteSpace: 'nowrap',
+        textOverflow: 'ellipsis',
+        ...divProps.style,
+      }}
```

Comment:
The style on main has changed this then.

Reply:
I don't see a good way to share the styles in a common component now that the name is a link, I feel like the component I would have to make is too abstract for just displaying the name of an item, do you think it is worth it?

Reply:
Yeah, I don't think those styles need to be part of this component, but I just want to make sure, that we don't regress on the changes on main since this was created.

Reply:
I made it so, that they share the same icon, I think that is the thing most likely to change